modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJavaCommerceCountryCheck.testjava

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+import com.liferay.commerce.model.CommerceAddress;
+import com.liferay.portal.kernel.model.Country;
+
+/**
+ * @author Nícolas Moura
+ */
+public class UpgradeJavaCommerceCountryCheck {
+
+	public void method(CommerceAddress commerceAddress) {
+		Country country = commerceAddress.getCountry();
+
+		// Replacement cases
+
+		country.getTwoLettersISOCode();
+
+		country.setTwoLettersISOCode();
+
+		country.getThreeLettersISOCode();
+
+		country.setThreeLettersISOCode();
+
+		// Chaining case
+
+		commerceAddress.getCountry().getThreeLettersISOCode();
+
+		// Chaining case with a breakline
+
+		commerceAddress.getCountry(
+		).setTwoLettersISOCode();
```

Comment:
@nicolas-sss, is it properly formatted according SF?

Reply:
yes, was the SF that break the line that way. I believe the format for chaining in general cases is like this:
```
commerceAddress.getCountry(
    parameter1,
    parameter2
).setTwoLettersISOCode();
```

As this case has no parameters it stayed that way:
```
commerceAddress.getCountry(
).setTwoLettersISOCode();
```